lib/signet/vm.ex

```diff
@@ -37,6 +37,22 @@ defmodule Signet.VM do
   @max_uint256 @two_pow_256 - 1
   @gas_amount 4_000_000
 
+  defmodule FFIs do
+    def logger_ffi(args) do
+      case Signet.Contract.ILogger.decode_call(args) do
+        {:ok, f, values} ->
+          IO.puts("Logger.#{f}: #{inspect(values, limit: :infinity, printable_limit: :infinity)}")
+        _ ->
+          nil
+      end
+      {:return, <<>>}
+    end
+  end
+
+  @builtin_ffis %{
+    ~h[0x0000000000000000000000000000000000FFCCCC] => &FFIs.logger_ffi/1
```

Comment:
`0xFF1000` instead? I reserved the first 10 addresses for framework-level FFIs.

Reply:
I now use forge's address, so the code can work identically in test and VM

Reply:
which is: `0x000000000000000000636F6e736F6c652e6c6f67`

Reply:
Ok that works too. Seems like it's the address that Hardhat uses as well